test/bin/check_ecs_service_health_spec.rb

```diff
@@ -3,6 +3,9 @@
 require_relative '../ecs_stubs.rb'
 
 describe 'CheckEcsServiceHealth' do
+  after :all do
+    Aws.config = {}
+  end
```

Comment:
this was a bug in this test file that left the AWS Config global object in a bad state. This cleaned it up.